resources/js/Pages/Collections/Components/Articles/ArticlesTab.tsx

```diff
@@ -0,0 +1,120 @@
+
+    const [displayType, setDisplayType] = useState(view === "list" ? DisplayTypes.List : DisplayTypes.Grid);
+
+    const [sortBy, setSortBy] = useState<ArticleSortBy>(
+        (sort in ArticleSortBy ? sort : defaults.sortBy) as ArticleSortBy,
+    );
+
+    const queryParameters: Record<string, string> = {
+        pageLimit: pageLimit.toString(),
+        sort: sortBy,
+        search: debouncedQuery,
+        view: displayType,
+    };
+
+    useEffect(() => {
+        replaceUrlQuery(queryParameters);
+    }, [JSON.stringify(queryParameters)]);
+
+    const { articles, isLoading } = useCollectionArticles(collection.slug, queryParameters);
+
+    if (isLoading || !isTruthy(articles)) {
+        return <>is loading</>;
+    }
+
+    const articlesCount = articles.paginated.meta.total;
+
+    return (
+        <div>
+            <div className="mb-3 mt-6 flex items-center justify-between gap-x-3 sm:mb-4">
+                <DisplayType
+                    displayType={displayType}
+                    onSelectDisplayType={(type) => {
+                        setDisplayType(type);
+                    }}
+                />
+
+                <div className="flex-1">
+                    <SearchInput
+                        className="hidden sm:block"
+                        placeholder={t("pages.collections.articles.search_placeholder")}
+                        query={query}
+                        onChange={(query) => {
+                            setQuery(query);
+                        }}
+                    />
+                </div>
+
+                <ArticleSortDropdown
+                    activeSort={sortBy}
+                    onSort={(sort) => {
+                        setSortBy(sort);
+                    }}
+                />
+            </div>
+            <div className="mb-4 sm:hidden">
+                <SearchInput
+                    placeholder={t("pages.collections.articles.search_placeholder")}
+                    query={query}
+                    onChange={(query) => {
+                        setQuery(query);
+                    }}
+                />
+            </div>
+
+            <div className="flex flex-col items-center space-y-6">
+                {articlesCount > 0 && displayType === DisplayTypes.Grid && (
+                    <ArticlesGrid articles={articles.paginated.data} />
+                )}
+
+                {articlesCount > 0 && displayType === DisplayTypes.List && (
+                    <ArticlesList articles={articles.paginated.data} />
+                )}
+
+                {articlesCount === 0 && query === "" && (
+                    <EmptyBlock>{t("pages.collections.articles.no_articles")}</EmptyBlock>
+                )}
+                {articlesCount === 0 && query !== "" && (
+                    <EmptyBlock>{t("pages.collections.articles.no_articles_with_filters")}</EmptyBlock>
+                )}
```

Comment:
the empty blocks should span the full width, currently they don't
<img width="1345" alt="image" src="https://github.com/ArdentHQ/dashbrd/assets/35610748/1a57b8f6-cf39-4239-99be-a8d7c5b0a626">